### PR TITLE
22 as an owner i can view the details of a delivery

### PR DESCRIPTION
**Pull request by juliusgambe**
- Change delivery command format
- Add get delivery by id to model manager

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/delivery/DeliveryViewCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+public class DeliveryViewCommandParser implements Parser<DeliveryViewCommand> {
+
+    private static final Pattern ARGUMENT_FORMAT = Pattern.compile(
+        "^(?<id>\\d+)$"
+    );
+
+    @Override
+    public DeliveryViewCommand parse(String args) throws ParseException {
+        if (args.isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```
Comment (B-enguin) [inline src/main/java/seedu/address/logic/parser/delivery/DeliveryViewCommandParser.java:25]: Perhaps add test to cover empty args

**Review by B-enguin — COMMENTED** [review]
General Feedback

**Review by zhonghan721 — APPROVED** [review]
LGTM

**Review by jianyangg — APPROVED** [review]
LGTM
